### PR TITLE
Prototype/auth-state

### PR DESCRIPTION
**Pull request by austinteshuba**
Handles authentication state changes, by showing/hiding the login button and features as needed. 

Signed Out:
Screenshot 2020-06-30 at 10 21 08 AM

Signed In:
Screenshot 2020-06-30 at 10 21 21 AM

Dependent on PR #17

### CODE REVIEW

File: src/main/webapp/auth.js [hunk 1]
```diff
@@ -35,12 +34,32 @@ function init() {
...
         });
   });
 }
 
+/**
+ * Will initialize the page based on the current authentication state
+ */
+function handleAuthenticationState() {
+  const featureContainer = document.querySelector('.feature-container');
+  const logInButton = document.querySelector('#google-sign-in-btn');
```
Comment (tttzach) [inline src/main/webapp/auth.js:49]: extreme nit: maybe stick with one of login, log-in or sign-in?
e.g. `... logInButton = ... sign-in ...` to `... loginButton = ... login ...`
If you do change this, taking off or adding a dash for `login` line 52 and 57 might help too.

Comment (austinteshuba) [reply src/main/webapp/auth.js:49]: Agreed! Good catch

**Comment by gbuenoandrade** [comment]
> Handles authentication state changes, by showing/hiding the login button and features as needed.

Please add `depends on #<PR num>` to the end of your description

**Comment by austinteshuba** [comment]
@gbuenoandrade Take a look again - it should appear. You may need a test account to use this (unless if you're okay using your @google acct.)

**Pull request merged**